ui/src/app/(index)/_components/UserInfoItem.tsx

```diff
@@ -0,0 +1,48 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import { Button } from '@/components/ui/button';
+
+const UserInfoItem = ({
+    alt,
+    src,
+    number,
+    show,
+    title,
+    description,
+    link,
+    text
+}: {
+    alt: string,
+    src: string,
+    number: number,
+    show: boolean,
+    title: string,
+    description: string,
+    link: string,
+    text: string
+}) => {
```

Comment:
I'm not a fan of this component having 10 props to pass in. One thing a lot of the reusable components do, for example our shadcn/ui components, is have components you can pass in as prop. So for example we could use this component like this
```
<UserInfoItem displayName="Admin" url="link-to-admin-page">
    <Image />
    <span className="text-[2.5rem] text-text-color">5</span>
</UserInfoItem>
```
The name and url props would how the Link button. renders